Firmware/Marlin_main.cpp

```diff
@@ -2820,15 +2820,15 @@ static void gcode_G80()
     run = false;
 #endif //PINDA_THERMISTOR
 
-    uint8_t nMeasPoints = code_seen('N') ? code_value_uint8() : eeprom_read_byte((uint8_t*)EEPROM_MBL_POINTS_NR);
-    if (nMeasPoints != 7) {
-        nMeasPoints = 3;
-    }
+    uint8_t nMeasPoints = eeprom_read_byte((uint8_t*)EEPROM_MBL_POINTS_NR);
+    bool bCodeSeen = code_seen('N');
+    if (uint8_t value = code_value_uint8(); bCodeSeen && (value == 7 || value == 3))
```

Comment:
ngl, this is some weird syntax, but if it results in more optimal code, no problem with this :)

Reply:
It should be legal in C++17 https://en.cppreference.com/w/cpp/language/if